test/relativistic/relativistic_time_dilation_test.py

```diff
@@ -0,0 +1,47 @@
+from collections import namedtuple
+from pytest import approx, fixture, raises
+from symplyphysics import (
+    errors,
+    units,
+    Quantity,
+    SI,
+    convert_to,
+)
+from symplyphysics.laws.relativistic import relativistic_time_dilation
+
+
+# Using calculations from the paper: https://www.omnicalculator.com/physics/time-dilation
+
+
+@fixture(name="test_args")
+def test_args_fixture():
+    t0 = Quantity(10 * units.second)
```

Comment:
Explain `t0`. Is it stationary or moving?

Reply:
But you pass it as rest time. So you are measuring the time on the spaceship.

Reply:
Ok, I will fix that. Variable naming on the law itself looks confusing as well.